Symfony/CS/Fixer/PSR2/OneLineMethodArgumentFixer.php

```diff
@@ -0,0 +1,208 @@
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, Tokens $tokens)
+    {
+        /** @var Token $token */
+        foreach ($tokens as $index => $token) {
+            if (!$token->equals('(')) {
+                continue;
+            }
+
+            if (
+                // function
+                $this->isFunction($tokens, $index - 3)
+                // callable
+                || $this->isFunction($tokens, $tokens->getPrevMeaningfulToken($index))
+                // function call
+                || $this->isFunction($tokens, $tokens->getPrevMeaningfulToken($index), T_STRING)
+            ) {
+                $endIndex = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $index);
+
+                if ($this->hasNewLineBetweenParenthesis($tokens, $index, $endIndex)) {
+                    $this->fixFunctionArguments($tokens, $index, $endIndex);
+                }
+            }
+        }
+    }
+
+    /**
+     * Fix arguments for given function.
+     *
+     * @param Tokens $tokens             Tokens to handle
+     * @param int    $startFunctionIndex Start parenthesis position
+     * @param int    $endFunctionIndex   End parenthesis position
+     */
+    private function fixFunctionArguments(Tokens $tokens, $startFunctionIndex, $endFunctionIndex)
+    {
+        for ($index = $startFunctionIndex; $index <= $endFunctionIndex; $index++) {
+            $token = $tokens[$index];
+
+            if ($token->equals('(')) {
+                if (!$this->isNewLine($tokens[$index + 1]) && $tokens[$index + 1]->getContent() !== ')') {
+                    $this->addNewLineOn($tokens, $index, $endFunctionIndex);
+                }
+            }
+            if ($token->equals(')')) {
+                if (!$this->isNewLine($tokens[$index - 1]) && $tokens[$index - 1]->getContent() !== '(') {
+                    $this->addNewLineOn($tokens, $index - 1, $endFunctionIndex, '');
+                }
+            }
+
+            if ($token->equals(',')) {
+                $this->fixNewLine($tokens, $index, $endFunctionIndex);
+            }
+        }
+    }
+
+    /**
+     * Inserts new line after comma.
+     *
+     * @param Tokens $tokens
+     * @param int    $index
+     * @param int    $endFunctionIndex
+     */
+    private function fixNewLine(Tokens $tokens, $index, &$endFunctionIndex)
+    {
+        if (
+            $tokens[$index + 1]->isWhitespace()
+            && $tokens[$index + 2]->isComment()
+            && $this->isNewLine($tokens[$index + 3])
+        ) {
+            $index += 3;
+        }
+
+        $this->addNewLineOn($tokens, $index, $endFunctionIndex);
+    }
+
+    /**
+     * Adds new line to the given token.
+     *
+     * @param Tokens $tokens
+     * @param int    $index
+     * @param int    $endFunctionIndex
+     * @param string $indention
+     */
+    private function addNewLineOn(Tokens $tokens, $index, &$endFunctionIndex, $indention = '    ')
```

Comment:
I believe you can use https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/1.11/Symfony/CS/Tokenizer/Tokens.php#L288 for this but I could be wrong :)